python/mlcroissant/mlcroissant/_src/datasets_test.py

```diff
@@ -192,13 +191,16 @@ def test_nonhermetic_loading(version, dataset_name, record_set_name, num_records
 # Non-hermetic test cases for croissant >=1.0 only (data from the internet).
 @pytest.mark.nonhermetic
 @pytest.mark.parametrize(
-    ["dataset_name", "record_set_name", "num_records"],
+    ["dataset_name", "record_set_name", "num_records", "filters"],
```

Comment:
So here we could also e.g. have 

`["huggingface-anthropic-hh-rlhf/metadata.json", "harmless-base", 10, "train"]`

right?

Reply:
I haven't tried this dataset yet!

Reply:
But potentially yes.